account/manager.go

```diff
@@ -188,16 +188,21 @@ func (m *Manager) start() error {
 	if err != nil {
 		return fmt.Errorf("unable to retrieve accounts: %v", err)
 	}
+
+	// We calculate the default fee rate that will be used
+	// for resuming accounts for which we haven't created and broadcast
+	// a transaction for yet
+	feeRate, err := m.cfg.Wallet.EstimateFee(
+		ctx, int32(DefaultFundingConfTarget),
+	)
+	if err != nil {
+		return fmt.Errorf("unable to estimate default fees for resuming"+
```

Comment:
nit: add a space at the end after `resuming` as otherwise it'll concatenate with the next line. Also I think the error itself could just be: "unable to estimate default fees". I personally like long errors but overall we tend to not overexplain error cases and try to reduce to the actual cause.